Project.toml

```diff
@@ -11,8 +11,9 @@ Distributed = "8ba89e20-285c-5b6f-9357-94700520ee1b"
 GraphRecipes = "bd48cda9-67a9-57be-86fa-5b3c104eda73"
 Graphs = "86223c79-3864-5bf0-83f7-82e725a168b6"
 JLD2 = "033835bb-8acc-5ee8-8aae-3f567f8a3819"
+LightOSM = "d1922b25-af4e-4ba3-84af-fe9bea896051"
```

Comment:
Don't forget to add a version of this in the `[compat]` otherwise registrator will complain.